pkgs/by-name/ne/nerdfetch/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitea
+}:
+stdenv.mkDerivation rec {
+  pname = "nerdfetch";
+  version = "8.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "thatonecalculator";
+    repo = "NerdFetch";
+    rev = "v${version}";
```

See above regarding switch to finalAttrs
```suggestion
    rev = "v${finalAttrs.version}";
```